packages/libs/pactjs-generator/src/contract/parsing/utils/parser-utilities/tests/asString.test.ts

```diff
@@ -0,0 +1,27 @@
+import { getPointer } from '../../getPointer';
+import { asString } from '../asString';
+import { block } from '../block';
+import { id } from '../id';
+import { FAILED } from '../rule';
+import { seq } from '../seq';
+import { skipTheRest } from '../skip';
+
+describe('asString parser', () => {
+  it('should return result as string if rule is matched', () => {
```

Comment:
we can make the `it`s without `should`.
```suggestion
  it('returns result as string if rule is matched', () => {
```

Idem for the rest